api.ts

```diff
@@ -488,11 +509,26 @@ export function isSyncPackFormula(fn: BaseFormula<ParamDefs, any>): fn is Generi
   return Boolean((fn as GenericSyncFormula).isSyncFormula);
 }
 
+/**
+ * The return value from the formula that implements a sync table. Each sync formula invocation
+ * returns one reasonable size page of results. The formula may also return a continuation, indicating
+ * that the sync formula should be invoked again to get a next page of results. Sync functions
+ * are called in a loop until there is no continuation returned.
```

Comment:
not sure if `in a loop` could be made more clear.

Reply:
Sure, changed it to "repeatedly", but happy to adjust the language as desired.